### ISSUE TITLE
Lastest build in npm doesn't include scss files

### ISSUE THREAD

**Issue by kblcuk** [JedWatson/react-select#2164]
Latest (1.0.0) release doesn't include scss files, so there is no way to include them in own scss build.
Basically doing something like this:
```
// Override react-select variables: https://github.com/JedWatson/react-select/blob/master/scss/select.scss
// TODO Bulma uses em/rem, figure out a good number here as well
//$select-input-height: 2.285em;
$select-input-height: 32px;
$select-input-internal-height: 28px;
$select-input-border-radius: 0;
$select-item-border-radius: 0;
@import "~react-select/scss/default";
```

Seems like many other things are missing as well:

```
$ npm i react-select
$ l node_modules/react-select
total 168
drwxr-xr-x   8 alex  wheel   272B Nov 23 11:46 .
drwxr-xr-x  22 alex  wheel   748B Nov 23 11:46 ..
-rw-r--r--   1 alex  wheel    44K Nov 23 11:23 HISTORY.md
-rw-r--r--   1 alex  wheel   1.1K Oct 27 08:42 LICENSE
-rw-r--r--   1 alex  wheel    24K Nov 23 06:50 README.md
drwxr-xr-x   7 alex  wheel   238B Nov 23 11:46 dist
drwxr-xr-x  10 alex  wheel   340B Nov 23 11:46 lib
-rw-r--r--   1 alex  wheel   5.0K Nov 23 11:46 package.json

$ npm i react-select@1.0.0-rc.10
$ l node_modules/react-select
total 744
drwxr-xr-x  32 alex  wheel   1.1K Nov 23 11:47 .
drwxr-xr-x  22 alex  wheel   748B Nov 23 11:47 ..
-rw-r--r--   1 alex  wheel   170B Sep 11 07:52 .babelrc
-rw-r--r--   1 alex  wheel    71B Jan 31  2017 .coveralls.yml
-rw-r--r--   1 alex  wheel    63B Jan 31  2017 .eslintignore
-rw-r--r--   1 alex  wheel   921B Sep  9 16:32 .eslintrc.js
-rw-r--r--   1 alex  wheel    50B Sep  9 15:58 .gitattributes
-rw-r--r--   1 alex  wheel    39B Jan 31  2017 .npmignore
-rw-r--r--   1 alex  wheel   171B Sep  9 15:58 .nycrc
drwxr-xr-x   9 alex  wheel   306B Nov 23 11:47 .publish
-rw-r--r--   1 alex  wheel   112B Sep  9 16:46 .travis.yml
-rw-r--r--   1 alex  wheel   1.5K Sep  9 15:58 CHANGES.md
-rw-r--r--   1 alex  wheel   3.2K Sep  9 15:58 CONTRIBUTING.md
-rw-r--r--   1 alex  wheel    37K Sep 13 16:21 HISTORY.md
-rw-r--r--   1 alex  wheel   1.2K Sep  9 15:58 ISSUE_TEMPLATE.md
-rw-r--r--   1 alex  wheel   1.1K Sep  9 17:03 LICENSE
-rw-r--r--   1 alex  wheel    22K Sep 13 16:21 README.md
-rw-r--r--   1 alex  wheel   1.1K Sep  9 15:58 TODO.md
drwxr-xr-x   7 alex  wheel   238B Nov 23 11:47 dist
drwxr-xr-x   3 alex  wheel   102B Jan 31  2017 examples
drwxr-xr-x   9 alex  wheel   306B Nov 23 11:47 less
drwxr-xr-x  10 alex  wheel   340B Nov 23 11:47 lib
-rw-r--r--   1 alex  wheel   1.0K Sep 11 17:24 package-scripts.js
-rw-r--r--   1 alex  wheel   4.7K Nov 23 11:47 package.json
-rw-r--r--   1 alex  wheel   1.1K Sep 12 11:47 rollup.config.js
drwxr-xr-x  10 alex  wheel   340B Nov 23 11:47 scss
drwxr-xr-x  11 alex  wheel   374B Nov 23 11:47 src
drwxr-xr-x   7 alex  wheel   238B Nov 23 11:47 test
drwxr-xr-x   4 alex  wheel   136B Nov 23 11:47 testHelpers
-rw-r--r--   1 alex  wheel   458B Sep  9 15:58 wallaby.js
-rw-r--r--   1 alex  wheel   1.4K Sep  9 16:22 webpack.config.js
-rw-r--r--   1 alex  wheel   229K Sep 12 11:48 yarn.lock
```

Is this intended change?

**Comment by hudochenkov** [comment]
We faced same problem. I believe this is the reason https://github.com/JedWatson/react-select/pull/2062

**Comment by vishaltelangre** [comment]
I downgraded to an older version temporarily to get things working on my project.

```diff
--- a/package.json
+++ b/package.json
@@ -15,7 +15,7 @@
-    "react-select": "^1.0.0",
+    "react-select": "^1.0.0-rc.10",
```

**Comment by hutchy2570** [comment]
We've had to downgrade to 1.0.0-rc.10 also as we rely on the .less files.

**Comment by hudochenkov** [comment]
@vishaltelangre you have to lock it without `^` otherwise it will be updated to `1.0.0`:

```
"react-select": "1.0.0-rc.10"
```

**Comment by vishaltelangre** [comment]
@hudochenkov Yes. Thanks.

**Comment by JedWatson** [comment]
Sorry about this - was indeed an unintended side effect of #2062

I've just published 1.0.1 with a fix

**Comment by hudochenkov** [comment]
@JedWatson Thank you!
